DQM/HcalMonitorTasks/src/HcalDigiMonitor.cc

```diff
@@ -578,6 +578,20 @@ void HcalDigiMonitor::analyze(edm::Event const&e, edm::EventSetup const&s)
 	if (debug_>0)
 		std::cout << "### Processing FED: " << i << std::endl;
 
+	//	For uTCA spigos are useless => by default we have digisize = 4
+	//	As of 20.05.2015
+	//	HF = 2
+	if ((i>=1118 && i<=1122) ||
+			(i>=718 && i<=723))
```

Comment:
are there no constants already defined corresponding to whatever "1118", "1122", "718", and "723" happen to be?